src/config/builder.rs

```diff
@@ -295,14 +295,12 @@ impl ConfigBuilder {
 
     #[cfg(test)]
```

Comment:
Know why there isn't an equivalent for yaml?

Reply:
Hmm, I'm not sure. There are definitely tests testing YAML deserialization; I guess no tests needed a helper for YAML though.